projects/packages/publicize/actions.php

```diff
@@ -8,12 +8,12 @@
 // If WordPress's plugin API is available already, use it. If not,
 // drop data into `$wp_filter` for `WP_Hook::build_preinitialized_hooks()`.
 if ( function_exists( 'add_action' ) ) {
```

Comment:
I'm happier with this approach as I can see a number of the other packages doing it in the same way, but I wonder if we should restrict this to just WPCOM, similar to the Connection package https://github.com/Automattic/jetpack/blob/38efec6d76d09c00e7d69dea2d2e40ad459cef6d/projects/packages/connection/actions.php#L8

It might again be something we do in a follow-up but this means we're circumventing the Config mechanisms. I'll add another comment in class-publicize-setup.php to explain what I mean

Reply:
Yeah, we can use that approach. We can do that in a follow up to unblock other tasks here.